src/app/config.rs

```diff
@@ -90,6 +94,12 @@ pub struct Config {
 
     /// Allow uptime checks against internal IP addresses
     pub allow_internal_ips: bool,
+
+    /// Name of the pod that is running
+    pub pod_name: String,
```

Comment:
wait do you need this, can't you just get `hostname`?

Reply:
What's this actually for

Reply:
Discussed on slack, will change to `checker_id`